package/gluon-ebtables-net-rules/files/lib/gluon/ebtables/110-net-allow-ipv6

```diff
@@ -0,0 +1,9 @@
+siteConfig = require("gluon.site_config")
+
+rule ('NET_SRC -p IPv6 --ip6-src fe80::/10 -j RETURN')
+rule ('NET_SRC -p IPv6 --ip6-dst ff00::/8 -j RETURN')
```

Comment:
It's just three, so it'd be three rules :-). One for 131, one for 135 and one for 143.